simtools/applications/simulate_showers_for_trigger_rates.py

```diff
@@ -124,6 +157,12 @@ def _parse(label=None, description=None):
     return config.initialize(simulation_model="telescope", job_submission=True, db_config=True)
 
 
+def print_list_into_file(list_of_files, file_name):
```

Comment:
Missing docstring.

Reply:
Yes, it it was before. Implemented the docstring now.